File: blackjack/Player.java
package blackjack;

import java.util.ArrayList;

public class Player {

    private ArrayList<Card> hand;
    private String name;

    public Player(String name) {
        this.hand = new ArrayList<Card>();
        this.name = name;
    }

    public ArrayList<Card> getHand() {
        return this.hand;
    }

    public void hit(Card card) {
        this.hand.add(card);
    }

    public void printHand() {
        System.out.print(name + " has: ");
        for (Card card : this.hand) {
            card.printCard();
            System.out.print(" ");
        }
    }

}

File: PythonBlackjack/Player.py
class Player:

    def __init__(self, name, money):
        self.name = name
        self.hand = []
        self.money = money
        self.bet = 0

    def setHand(self, hand):
        self.hand = hand

    def printHand(self):
        print(self.name, ": [", sep="", end=" ")
        for card in self.hand:
            card.printCard()
        print("]", end="")

    def placeBet(self, bet):
        self.bet = bet
        self.money -= bet

    def addMoney(self, money):
        self.money += money

    def printPlayerInfo(self):
        print(self.name, " ($", self.money, ")", sep="")
